### ISSUE TITLE
models worked in 0.9.0 doesn't work in 0.9.2

### ISSUE THREAD

**Issue by NightLancer** [Rist8/MiSide-UniversalAssetLoader#1]
I'm trying to use same .fbx that worked perfectly well for default Mita but it doesn't work anymore(no matter if I using it as is [with 0.01 scale] or multiply it by 100 and scale down later in the code)
![зображення](https://github.com/user-attachments/assets/9f9703ae-c31d-4c03-8e24-3a8c1934851d)

Code that worked before:

```
*UnderboobSweater
replace_mesh Mita Sweater untitled2 Sweater
```

It doesn't work anymore, as well as

```
*UnderboobSweater
replace_mesh Mita Sweater untitled2 Sweater
set_scale Mita Sweater 0.01 0.01 0.01
```
(I also tried adding extra `Mita` at the end but it doesnt matter since it's for default one anyway) 

What changed? I can get "mesh data size" change because of model size change - but why vertex stride is broken too? 
Would be nice to know how it fix this/update previous model for it. 'cause 0.9.2 seemed to have exact update I needed - but it just doesn't work for me-_-

**Comment by Rist8** [comment]
Yeah i see now, bad push, will rollback the file for AssetLoader.

**Comment by Rist8** [comment]
Updated 0.9.2 release so it has the old version of AssetLoader.cs

**Comment by NightLancer** [comment]
> Updated 0.9.2 release so that it has the old version of AssetLoader.cs

thanks!

Also I had one side-question: is it possible to implement folders loading in for the addons_config? To have something like this:
![зображення](https://github.com/user-attachments/assets/fb674b0a-a71f-4610-aa85-cea78209fff0)

But if it's too bug-inducing(I know how using _path_ instead of string can complicate things - I've dealt with that in Python-_-) - then no big deal. Just kinda wanted to prevent over-cluttering of files in [potential] case if game gets many mods. (it's not as crucial for now with scarce mod's quantity)

**Comment by Rist8** [comment]
Implemented this feature in 0.9.3 version
